platform/model/bucket_aggregations/terms.go

```diff
@@ -245,3 +250,27 @@ func CheckParamsTerms(ctx context.Context, paramsRaw any) error {
 
 	return nil
 }
+
+func (query Terms) NewRowsTransformer() model.QueryRowsTransformer {
+	return &TermsRowsTransformer{minDocCount: int64(query.minDocCount)}
+}
+
+type TermsRowsTransformer struct {
+	minDocCount int64
+}
+
+func (qt TermsRowsTransformer) Transform(ctx context.Context, rowsFromDB []model.QueryResultRow) []model.QueryResultRow {
```

Comment:
I saw this code before, any chance to unify that logic.

Reply:
Good idea, adding as TODO, as merging this PR asap would help me in some other ones.